orchagent/fdborch.cpp

```diff
@@ -1056,6 +1061,11 @@ void FdbOrch::flushFDBEntries(sai_object_id_t bridge_port_oid,
         attr.value.oid = vlan_oid;
         attrs.push_back(attr);
     }
+    
+    /* do not flush static mac */
+    attr.id = SAI_FDB_FLUSH_ATTR_ENTRY_TYPE;
```

Comment:
With this change added here it means for any "FDB flush" operations whether it is based on port, or VLAN it will never flush out Static MACs.  Not sure if this may impact MLAG operations where they may want to flush out remotely learnt MAC that were programmed as Static MAC?  Adding MLAG owner @Praveen-Brcm to the review to ensure this is ok.